src/Orchard.Cms.Web/Modules/Orchard.OpenId/Models/OpenIdScope.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Orchard.OpenId.Models
+{
+    public class OpenIdScope
```

Comment:
Ia there a scenario where this classes could be used outside of this module? Like by a companion module? 
This would require a separate `.Abstractions` project.